Angular2WebpackVisualStudio/src/Angular2WebpackVisualStudio/webpack.prod.js

```diff
@@ -84,18 +85,17 @@ module.exports = {
         }),
         new webpack.optimize.CommonsChunkPlugin(
             {

```

Comment:
so we leave the chunking as it for both dev and prod and add the tslint
@afnpires Can you update the PR and we can integrate?

I do not want to use the angular-cli plugin

Thanks all